kibana/config/kibana.yml

```diff
@@ -4,10 +4,20 @@
 #
 server.name: kibana
 server.host: 0.0.0.0
-elasticsearch.hosts: [ "http://elasticsearch:9200" ]
+elasticsearch.hosts: [ "http://elasticsearch:9200" ]  # (!) set protocol to https when TLS is enabled
```

Comment:
@deviantony to your comment in https://github.com/deviantony/docker-elk/pull/495#issuecomment-673165902, multiplying configurations to support multiple variants turns out to result in a confusing experience, especially with the multiple pre-configured extensions we support.

Here's a crazy idea for you: would it be bad if we un-commented all the commented lines from this PR, and just enabled TLS by default? 🙃 It's 2020, TLS is a default almost everywhere, and addressing ES over `https` is not a significant change from a user's perspective.

Reply:
Well on the user experience side that would be great but that would open up to the concerns you stated in the following comments:

* https://github.com/deviantony/docker-elk/issues/402#issuecomment-571152166 (point 1)
* https://github.com/deviantony/docker-elk/pull/495#issuecomment-706693683 (point 2)

Reply:
Alright fair enough, I think I've run out of ideas then. I'll close that story for now because I can't really dedicate time to it.